<launch>

    <!-- 2D / 3D Recognition Arg -->
    <arg name="gesture_recognition_3D" default="True"/>

    <!-- Recognition Precision Arg / Param -->
    <arg   name="recognition_precision_probability" default="0.8"/>
    <param name="recognition_precision_probability" value="$(arg recognition_precision_probability)"/>

    <!-- Mediapipe 2D Gesture Recognition Node -->
    <node unless="$(arg gesture_recognition_3D)" pkg="mediapipe_gesture_recognition" type="recognition_node_2D.py" name="recognition_node_2D" output="screen"/>

    <!-- Mediapipe 3D Gesture Recognition Node -->
    <node if="$(arg gesture_recognition_3D)"     pkg="mediapipe_gesture_recognition" type="recognition_node_3D.py" name="recognition_node_3D" output="screen"/>

</launch>
